src/mlpack/methods/ann/layer/instance_norm.hpp

```diff
@@ -0,0 +1,214 @@
+/**
+ * @file methods/ann/layer/instance_norm.hpp
+ * @author Anjishnu Mukherjee
+ *
+ * Definition of the Instance Normalization layer class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_INSTANCENORM_HPP
+#define MLPACK_METHODS_ANN_LAYER_INSTANCENORM_HPP
```

Comment:
``
```suggestion
#ifndef MLPACK_METHODS_ANN_LAYER_INSTANCE_NORM_HPP
#define MLPACK_METHODS_ANN_LAYER_INSTANCE_NORM_HPP
```

Just to be consistent.